modules/loader.js

```diff
@@ -426,7 +426,7 @@ __private.loadBlockChain = function () {
 					library.config.loading.snapshot = (round > 1) ? (round - 1) : 1;
 				}
 
-				modules.rounds.setSnapshotRounds(library.config.loading.snapshot);
+				//modules.rounds.setSnapshotRounds(library.config.loading.snapshot);
```

Comment:
Because `rounds` module was removed. I will adjust logic later to handle snapshoting properly.